selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/webdriver/SeleniumWebDriverHelper.java

```diff
@@ -953,6 +954,31 @@ public boolean isVisible(WebElement webElement) {
     }
   }
 
+  /**
+   * Determines whether or not this element {@link WebElement} is selected (uses for checkboxes,
+   * radio buttons, etc.)
+   *
+   * @param webElement element which should be selected
+   * @return true if given element is selected
+   */
+  public boolean isSelected(WebElement webElement) {
+    try {
+      return webElement.isSelected();
```

Comment:
Looks like `waitVisibility(webElemend)` command is missed.

Reply:
From the other hand, this method will duplicate existed `waitElementIsSelected()` method.

Reply:
The methods have different return values, so they are not duplicated.

Reply:
In that case I would say that this method is useless because it's simple to call `webElement.isSelected()` instead.

Reply:
I missed the method had ignored exception if element is not existed. It's weird behavior to ignore ebsence of element, IMHO.

Reply:
Seems, you are right, I delete this method from the 'SeleniumWebDriverHelper' as useless.